build/dependencies.props

```diff
@@ -1,31 +1,53 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <AspNetCoreVersion>2.1.0-*</AspNetCoreVersion>
-    <CoreFxVersion>4.4.0-*</CoreFxVersion>
-    <DiagnosticSourceVersion>4.4.1-*</DiagnosticSourceVersion>
-    <DependencyModelVersion>2.0.0-*</DependencyModelVersion>
-    <InternalAspNetCoreSdkVersion>2.1.1-*</InternalAspNetCoreSdkVersion>
-    <JsonNetVersion>10.0.1</JsonNetVersion>
-    <NETStandardImplicitPackageVersion>2.0.0-*</NETStandardImplicitPackageVersion>
-    <NETStandardLibraryNETFrameworkVersion>2.0.0-*</NETStandardLibraryNETFrameworkVersion>
-    <RelinqVersion>2.2.0-alpha-002</RelinqVersion>
-    <RoslynVersion>2.3.1</RoslynVersion>
-    <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
-    <SQLitePCLRawVersion>1.1.8</SQLitePCLRawVersion>
-    <StyleCopAnalyzersVersion>1.0.0</StyleCopAnalyzersVersion>
-    <SystemCollectionsImmutableVersion>1.4.0-*</SystemCollectionsImmutableVersion>
-    <SystemInteractiveAsyncVersion>3.1.1</SystemInteractiveAsyncVersion>
-    <TestSdkVersion>15.3.0</TestSdkVersion>
-    <XunitVersion>2.3.0-beta4-build3742</XunitVersion>
-    <XunitVersionInSpecProjects>2.2.0</XunitVersionInSpecProjects>
+    <MicrosoftEntityFrameworkCoreVersion>3.1.2</MicrosoftEntityFrameworkCoreVersion>
```

Comment:
For personal projects I totally agree. But we have 6 projects here that all need to use the same version, and different people might work on different aspects of the solution (with random contributors from time to time), which can quickly lead to version inconsistencies, if they are spread over multiple files.
Because of that, we actually block the compilation process in Pomelo if there is even a single version hard coded in a project file. Arcade does the same out-of-the-box and it generally makes sense to use EF Core itself (or Npgsql) as the general guideline.